### PR TITLE
Enhance table with ajax

### PR DESCRIPTION
**Pull request by boxed**

### CODE REVIEW

File: iommi/style_base.py [hunk 1]
```diff
@@ -79,6 +85,8 @@
     ),
     Query=dict(
         template='iommi/query/form.html',
+        assets__ajax_enhance__template='iommi/table/ajax_enhance.html',
```
Comment (jlubcke) [inline iommi/style_base.py:88]: Would it not be more natural to have it at `iommi/query/ajax_enhance.html`

Comment (boxed) [reply iommi/style_base.py:88]: Or on `Table`...

Comment (jlubcke) [reply iommi/style_base.py:88]: But it is only needed when we filter, right? When just using Table on a list of objetcs this is not a thing.

Comment (boxed) [reply iommi/style_base.py:88]: True. And I guess we could tweak it so it can work on a non-table Query target too in the future when the need arrives.

**Pull request merged**